pkg/kubelet/container/helpers.go

```diff
@@ -333,22 +333,18 @@ func MakePortMappings(container *v1.Container) (ports []PortMapping) {
 			}
 		}
 
-		// We need to create some default port name if it's not specified, since
-		// this is necessary for the dockershim CNI driver.
-		// https://github.com/kubernetes/kubernetes/pull/82374#issuecomment-529496888
-		if p.Name == "" {
-			pm.Name = fmt.Sprintf("%s-%s-%s:%d", container.Name, family, p.Protocol, p.ContainerPort)
-		} else {
-			pm.Name = fmt.Sprintf("%s-%s", container.Name, p.Name)
```

Comment:
simplifying code review: port mapping name used to be needed, but as dockershim has moved to use CNI, name is only used locally and thus don't need the container name to be part of it. This method is scoped for a single container. PR where the hostport functionality was removed from DockerShim to CNI: #62212